app/repositories/TodoRepository.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace App\Repositories;
+
+use App\Models\Todo;
+use App\Entities\TodoEntity;
+use Illuminate\Auth\AuthManager;
+
+
+class TodoRepository
+{
+    private AuthManager $authService;
+    public function __construct(AuthManager $authService)
+    {
+        $this->authService = $authService;
+    }
+    public function getAll()
+    {
+        $todos = $this->authService->user()->todos()->get();
```

Comment:
As we won't have the auth service in the repo. this should be replaced with something like

```php
public function getAllByUser(int $userId) {
    return Todos::where('user_id', $userId)->get();
}
```